shared/src/commonTest/kotlin/co/touchlab/kampkit/BreedModelTest.kt

```diff
@@ -80,6 +83,29 @@ class BreedModelTest : BaseTest() {
         }
     }
 
+    @OptIn(ExperimentalTime::class)
+    @Test
+    fun updateDatabaseTest() = runTest {
```

Comment:
I don't see exactly why (might be too late at night) but this test still passes even if I revert the change in #150 so I think somethings not quite right.

Reply:
It was because of an off by one ms and copy paste issue cancelling each other our to cause the test to not actually test the change